themes/ddbasic/gulpfile.js

```diff
@@ -39,15 +39,31 @@ gulp.task('uglify', 'Minify JavaScript using Uglify',
 
 gulp.task('sass', 'Process SCSS using libsass',
   function () {
+    var includePaths = [
+      'node_modules/compass-mixins/lib',
+      // Zen grids is downloaded as a library using drush make.
+      '../../libraries/zen-grids/stylesheets'
```

Comment:
The problem with this is that we're using version 1.4 of zen-grids and the only versions available through npm 2.0.0 and up. It will therefore probably require some adjustment to the sass.